main.go

```diff
@@ -113,18 +102,27 @@ func subsystem() ([]cgroups.Subsystem, error) {
 	return s, nil
 }
 
-type cgroupMetrics struct {
-	*cgroups.Metrics
+type cgroupV1Metrics struct {
+	*v1.Metrics
 	Process *ProcessStats `json:"process_stats"`
 }
 
-func statsCgroups(ctx context.Context, system cgroups.Cgroup) (map[string]*cgroupMetrics, error) {
+func statsCgroupsV1(ctx context.Context) (map[string]*cgroupV1Metrics, error) {
+	// should we support hybrid here ?
+	if cgroups.Mode() != cgroups.Legacy {
+		return map[string]*cgroupV1Metrics{}, nil
+	}
```

Comment:
yes, shoud we support hybrid version here.
```suggestion
	if cgroups.Mode() == cgroups.Unified {
		return map[string]*cgroupV1Metrics{}, nil
	}
```